usaspending_api/api_contracts/contracts/v2/disaster/recipient/business_category/spending.md

```diff
@@ -82,8 +82,16 @@ This endpoint provides insights on the recipient business categories which recei
     + Members
         + `desc`
         + `asc`
-+ `sort` (optional, string)
-    Optional parameter indicating what value results should be sorted by. Valid options are any of the fields in the JSON objects in the response. Defaults to the first field provided.
++ `sort` (optional, enum[string])
```

Comment:
Based on the [latest mock](https://bahdigital.invisionapp.com/share/SCIAEAG6G4J#/296042275_COVID-_Recipients_-_Recipient_Location_-_Recipient_Type), I don't think we need `geo_layer` as a required param in this file?

Reply:
removed